File: src/main/java/io/kuzzle/sdk/enums/Event.java
package io.kuzzle.sdk.enums;

/**
 * The enum Event type.
 */
public enum Event {
  disconnected,
  reconnected,
  connected,
  error,
  jwtTokenExpired,
  loginAttempt,
  offlineQueuePush,
  offlineQueuePop
}

File: src/main/java/io/kuzzle/sdk/enums/Event.kt
package io.kuzzle.sdk.enums

/**
 * The enum Event type.
 */
enum class Event {
    disconnected,
    reconnected,
    connected,
    error,
    tokenExpired,
    loginAttempt,
    offlineQueuePush,
    offlineQueuePop
}
